Twig/Extension/SonataAdminExtension.php

```diff
@@ -87,7 +112,13 @@ protected function getTemplate(FieldDescriptionInterface $fieldDescription, $def
             $template = $environment->loadTemplate($defaultTemplate);
 
             if (null !== $this->logger) {
-                $this->logger->warning(sprintf('An error occured trying to load the template "%s" for the field "%s", the default template "%s" was used instead: "%s". ', $templateName, $fieldDescription->getFieldName(), $defaultTemplate, $e->getMessage()));
+                $this->logger->warning(sprintf(
+                    'An error occured trying to load the template "%s" for the field "%s", '.
+                    'the default template "%s" was used instead.',
+                    $templateName,
+                    $fieldDescription->getFieldName(),
+                    $defaultTemplate
+                ), array('exception' => $e));
```

Comment:
Don't know if there is a concrete case for this. Maybe someone else could review that?